libethash-cl/kernels/cl/ethash.cl

```diff
@@ -236,10 +236,20 @@ do { \
 
 #endif
 
+typedef struct {
```

Comment:
If not possible to use the same definition as in C++, at least add a comment that this struct must match the other `SearchResult` one.

Reply:
It would be nice, but I don't think opencl undestands uint32_t. I'll try creating a common .h file for it than uses 'unsigned'.